Makefile.toml

```diff
@@ -419,6 +430,15 @@ exit_on_error true
 generate_generic example js lib/api
 '''
 
+[tasks.gen-demo-feature]
```

Comment:
this should be a part of `gen` too

Reply:
It should be already, it's under `gen-feature`